hack/ci/e2e.sh

```diff
@@ -187,9 +187,30 @@ EOF
         | grep -cv "node-role.kubernetes.io/master" \
     )"
 
-    # ginkgo regexes
-    SKIP="${SKIP:-}"
-    FOCUS="${FOCUS:-"\\[Conformance\\]"}"
+    # ginkgo regexes, use CI k/k presubmit regex by default
+    SKIP="${SKIP:-"\\[Slow\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]"}"
+    FOCUS="${FOCUS:-""}"
```

Comment:
/hold
explicitly holding this because conformance testing jobs at minimum should explicitly set focus / no skip and ignore any choices here.

this PR is useful to explore the tests, but we'll need to implement a bit differently to avoid suddenly changing what all of the various jobs using this script do.

Reply:
sure, I just was trying to get the right regex and identify the problems

Reply:
right, but this is a reminder that we should make sure not to merge it, as that will break the conformance CI :sweat_smile: